Documentation/network/concepts/ipam/index.rst

```diff
@@ -39,3 +39,10 @@ Dynamic CIDR/IP allocation   ❌                     ❌                       
    gke
    crd
    deep_dive
+
+.. note::
+
```

Comment:
How does this render?

Again I think this is a case where `.. note::` is unsuitable. Either create a section title for modifying IPAM configuration or just embed the text directly without a note.